nihms-native-assembler/src/main/java/org/dataconservancy/nihms/assembler/nihmsnative/NihmsZippedPackageStream.java

```diff
@@ -71,4 +72,13 @@ public StreamingSerializer getMetadataSerializer() {
     public void setMetadataSerializer(StreamingSerializer metadataSerializer) {
         this.metadataSerializer = metadataSerializer;
     }
+
+    public static String getNonCollidingFilename(String fileName, DepositFileType fileType) {
```

Comment:
Could this unit test coverage in `NihmsPackageStreamTest` along with some javadoc?

Reply:
Sure.